examples/kyb-app/src/components/layouts/AppShell/Navigation.tsx

```diff
@@ -7,7 +7,8 @@ import { useCallback } from 'react';
 export const Navigation = () => {
   const { state } = useDynamicUIContext();
   const { stateApi } = useStateManagerContext();
-  const isDisabled = state.isLoading;
+  const isFirstStep = !stateApi.getNextEvents()?.includes('PREVIOUS');
```

Comment:
Consider using Page Resolver context and check page is first by accessing `number` property and validating that its `1`